src/core/tests/frontend/paddle/test_models/gen_scripts/generate_elementwise_ops.py

```diff
@@ -162,6 +162,27 @@ def elementwise_pow(name : str, x, y, axis, in_dtype):
 
     return outs[0]
 
+def elementwise_floordiv(name : str, x, y, axis, in_dtype):
+    import paddle
+    paddle.enable_static()
+
+    with paddle.static.program_guard(paddle.static.Program(), paddle.static.Program()):
+        node_x = paddle.static.data(name = 'x', shape = x.shape, dtype = in_dtype)
+        node_y = paddle.static.data(name = 'y', shape = y.shape, dtype = in_dtype)
+        out = paddle.fluid.layers.nn.elementwise_floordiv(node_x, node_y, axis=axis)
```

Comment:
Please avoid use fluid api and use paddle 2.1 api instead. fluid is obsolete since paddle 1.8.
Could you help to check whether there is another elementwise_floordiv based OP in Paddle 2.1? if not, maybe we can still use the fluid one.

Reply:
paddle.fluid.layers.nn.elementwise_floordiv(x, y, axis=-1)  (paddle1.8 support axis = [0, last_x_dims]) --- > paddle.floor_divide(x, y, name=None)  (paddle2.x, only support axis = -1)

Reply:
paddle.floor_divide replace paddle.fluid.layers.nn.elementwise_floordiv in paddle2.x,  do you want to use paddle.floor_divide test in this case ?

Reply:
Add tests case for paddle.fluid.layers.nn.elementwise_floordiv, paddle.floor_divide.  elementwise_floordiv can work good.

Reply:
Great!